governance/default.go

```diff
@@ -642,10 +643,7 @@ func (g *Governance) getGovernanceCache(num uint64) (map[string]interface{}, boo
 
 func (g *Governance) addGovernanceCache(num uint64, data GovernanceSet) {
 	// Don't update cache if num (block number) is smaller than the biggest number of cached block number
-	g.idxCacheLock.Lock()
-	defer g.idxCacheLock.Unlock()
```

Comment:
Removing the lock here would change the original behavior, as line 654-567 won't be protected by the lock anymore. Is it okay?

Reply:
@ehnuje Instead, the operation moved to a new function `isNewBlockToAdd`. That part is to check if it is good to proceed or not. And `isNewBlockToAdd` is doing that work. So original behavior was not changed.

Reply:
@ehnuje My bad. It is changed. I think the last commit have to be rolled back.